product_tier_validation/models/product_template.py

```diff
@@ -1,36 +1,10 @@
 # Copyright 2021 Open Source Integrators
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import api, models
+from odoo import models
 
 
 class ProductTemplate(models.Model):
     _name = "product.template"
     _inherit = ["product.template", "tier.validation"]
     _tier_validation_manual_config = False
-
-    @api.model
-    def _is_tier_validated_active(self, state_code):
-        return state_code != "draft"
-
-    @api.model
-    def create(self, vals):
-        new = super().create(vals)
-        if new.need_validation and self._is_tier_validated_active(new.state):
```

Comment:
this will change existing expected behavior, no?
I don't care that much... just raising a possible issue :wink:

Reply:
aaah nevermind #948 does that...

Reply:
IIUC The activate/deactivate is now in another module by #948.

Reply:
Yes, that code was done by @dreispt in order to have that behaviour we have in product_state.